llvm/test/MC/AArch64/FP8_SME2/lut-diagnostics.s

```diff
@@ -25,3 +25,13 @@ luti4   {z0.b - z12.b}, zt0, {z0-z1}
 // CHECK: [[@LINE-1]]:{{[0-9]+}}: error: invalid number of vectors
 // CHECK-NEXT: luti4   {z0.b - z12.b}, zt0, {z0-z1}
 // CHECK-NOT: [[@LINE-1]]:{{[0-9]+}}:
+
```

Comment:
looks to me like adding a test for zip, inside this file breaks the convention, as zip is just part of SME2, so probably should be inside SME2 directory and use appropriate -mattr values